backend/src/main/java/io/debezium/configserver/rest/client/KafkaConnectClient.java

```diff
@@ -46,6 +47,11 @@
     @Produces("application/json")
     ConnectConnectorStatusResponse getConnectorStatus(@PathParam("connector-name") String connectorName) throws ProcessingException, IOException;
 
+    @GET
+    @Path("/connectors/{connector-name}/config")
+    @Produces("application/json")
+    Map<String,String> getConnectorConfig(@PathParam("connector-name") String connectorName) throws ProcessingException, IOException;
```

Comment:
Missing comma (see CI error).

```suggestion
    Map<String,String> getConnectorConfig(@PathParam("connector-name"), String connectorName) throws ProcessingException, IOException;
```

Reply:
the error was referring to the lack of spacing in Map<String,String>.  Will fix that

Reply:
Ah, yes. Sorry!